### PR TITLE
Release/0.9.2

### PR DESCRIPTION
**Pull request by colmsnowplow**
This PR removes the splitting of scripts by ; characters. Instead, we attempt to run the entire script, which can contain multiple statements.

This has the unhappy side effect of causing the BQ api to sometimes fail with success - where any but the last statement hits an error.

I believe this to be a bug in the API, since the query does actually fail, and once the error is retrieved it is of a different format to elsewhere (suggesting that it's potentially been handled for others where it's not handled in this case - will raise an issue with them once I've had a chance to produce a clean repro).

This is handled in our code by explicitly checking for an error after the query is complete, before moving on to the next query.

Additionally, we set Client.Location via the `region` variable - to mitigate issues which can occur due to BQ sometimes failing to correctly infer region

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,14 +1,13 @@
 module github.com/snowplow/sql-runner
 
-go 1.13
+go 1.15
```
Comment (colmsnowplow) [inline go.mod:3]: @jbeemster I meant to ask about this - any issues with bumping golang version? Can revert back to 1.13 if so. :)

Comment (jbeemster) [reply go.mod:3]: No issue from me - assuming everything works still :)

**Review by paulboocock — APPROVED** [review]
LGTM

Can we also update the broken images in the README? 😄

**Comment by colmsnowplow** [comment]
@jbeemster rc asset confirmed to be all good on prod. Can I get a green tick for release tomorrow?

**Review by jbeemster — APPROVED** [review]

**Pull request merged**